xapian-letor/tests/err_output21.txt

```diff
@@ -0,0 +1,2 @@
+Ranking score for qid:1 = -nan
+Average ranking score = -nan
```

Comment:
Produced by a test case which one qrel was missing

Reply:
Which is that test case there is nowhere in the whole file that err_output21 is being created??.

Reply:
Was commented out, force push to be deleted.